packages/widget_toolkit_pin/README.md

```diff
@@ -0,0 +1,270 @@
+```bash  
+$ flutter pub add widget_toolkit_pin widget_toolkit_biometrics  
+```
+
+Step 2: Follow the [integration instructions for the local_auth package](https://pub.dev/packages/local_auth)
+in order for the plugin to work on your desired platforms.
+
+Step 3: Pass the `WidgetToolkitTheme`,`PinCodeTheme` extensions to the `ThemeData` of your app:
+```dart
+MaterialApp(  
+  theme: ThemeData.light().copyWith(  
+    colorScheme: ColorScheme.fromSwatch(),
+      PinCodeTheme.light,
+      WidgetToolkitTheme.light,  
+    ],  
+  ),  
+  darkTheme: ThemeData.dark().copyWith(  
+    colorScheme: ColorScheme.fromSwatch(),  
+    extensions: [
+      PinCodeTheme.dark,
+      WidgetToolkitTheme.dark,
+    ],
+  ),  
+);
+```
+To access the content of the `widget_toolkit_pin`, you should import the files with the following line:
+```dart
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+```
+
+**Note:** The `WidgetToolkitTheme` comes from the **Widget Toolkit** package which already presence in the **Widget Toolkit Pin** package.
+It can be imported with the following line:
+```dart
+import 'package:widget_toolkit/widget_toolkit.dart';
+```  
+
+Step 3: Create an implementation of `BiometricsLocalDataSource`, for example:
+```dart
+class ProfileLocalDataSource implements BiometricsLocalDataSource { 
+	static const _areBiometricsEnabled = 'areBiometricsEnabled';
+	
+	Future<SharedPreferences> get _storageInstance => SharedPreferences.getInstance();
+	
+	@override  
+	Future<bool> areBiometricsEnabled() async {  
+	  final storage = await _storageInstance;  
+	  return storage.getBool(_areBiometricsEnabled) ?? false;  
+	}
+	
+	@override  
+	Future<void> setBiometricsEnabled(bool enable) async {  
+	  final storage = await _storageInstance;  
+	  await storage.setBool(_areBiometricsEnabled, enable);  
+	}
+}
+```
+
+Step 6: Create an implementation of `PinCodeService`. In this example we use double encryption. The 
+pin code is first encrypted on application level and then the encrypted value is again encrypted on
+operating system level, by using the `FlutterSecureStorage` instance. In your implementation, you 
+are free to choose the types of encryption. In the example two other packages are used:
+[encrypt](https://pub.dev/packages/encrypt) and [flutter_secure_storage](https://pub.dev/packages/flutter_secure_storage)
+In order for the `flutter_secure_storage` plugin to work on your desired platforms, follow the 
+integration instructions.
+```dart
+class AppPinCodeService implements PinCodeService {
+  static const _isPinCodeInStorage = 'pinCode';
+
+  FlutterSecureStorage get flutterSecureStorage => const FlutterSecureStorage();
+
+  @override
+  Future<bool> isPinCodeInSecureStorage() async {
+    var isPinCodeInSecureStorage =
+    await flutterSecureStorage.read(key: _isPinCodeInStorage);
+
+    if (isPinCodeInSecureStorage == null) {
+      return Future.value(false);
+    }
+    return Future.value(true);
```

Comment:
We can reduce this code block:

```dart
if (isPinCodeInSecureStorage == null) {
    return Future.value(false);
}
return Future.value(true);
```

to this:

```dart
return isPinCodeInSecureStorage != null;
```

Reply:
I have changed the code with the line.